cmd/sysinfo/sysinfo.go

```diff
@@ -64,68 +80,177 @@ func NewSysinfoCmd() *cobra.Command {
 	flags.BoolVar(&sysinfoSpec.ControllerRoleEnabled, "controller", true, "Include controller-specific sysinfo")
 	flags.BoolVar(&sysinfoSpec.WorkerRoleEnabled, "worker", true, "Include worker-specific sysinfo")
 	flags.StringVar(&sysinfoSpec.DataDir, "data-dir", constant.DataDirDefault, "Data Directory for k0s")
+	flags.StringVarP(&outputFormat, "output", "o", "human", "Output format. Must be one of human|yaml|json")
```

Comment:
This is how it's done in similar flags on other subcommands.

```suggestion
	flags.StringVarP(&outputFormat, "output", "o", "human", "Output format (valid values: human, json, yaml)")
```